Documentation/ReleaseNotes.md

```diff
@@ -33,6 +33,8 @@ For the smoothest upgrade path, please use the following steps.
     - MixedRealityToolkit
     - MixedRealityToolkit.Examples
     - MixedRealityToolkit.Extensions
+    > [!Note]
+    > If additional extensions have been installed, please take care when deleting this folder.
```

Comment:
I don't think we should just say to take care when deleting the folder right? i.e. we need to be more specific than "take care". Basically if folks have already copied over extensions into this folder, to back up this folder, and then re-import the extensions after updating to the latest version

Reply:
reworded, thanks!